src/wallet/wallet_tx_builder.cpp

```diff
@@ -628,3 +628,35 @@ TransactionBuilderResult TransactionEffects::ApproveAndBuild(
     // Build the transaction
     return builder.Build();
 }
+
+// TODO: Lock Orchard notes (#6226)
+void TransactionEffects::LockSpendable() const
```

Comment:
Suggestion: pass in the wallet reference rather than using the `pwalletMain` global.

Reply:
Yeah, done. Which made me rebase this on top of #6504, which has the `wallet` changes we talked about the other day.